app/src/test/java/protect/card_locker/DatabaseTest.java

```diff
@@ -291,14 +291,14 @@ public void addRemoveOneGroup() {
         assertTrue(result);
         assertEquals(1, DBHelper.getGroupCount(mDatabase));
 
-        Group group = DBHelper.getGroup(mDatabase, "group one");
+        Group group = DBHelper.getGroupByName(mDatabase, "group one");
```

Comment:
We should also test if we can get a group by ID, as that's the actual canonical thing.